blockchain/blockdao.go

```diff
@@ -52,6 +61,8 @@ var (
 	heightPrefix             = []byte("he.")
 	actionFromPrefix         = []byte("fr.")
 	actionToPrefix           = []byte("to.")
+	blockHeightToFileKey     = []byte("bhf.")
```

Comment:
I wonder if it's necessary two mappings? block and receipt of the same height should be in the same file.

Reply:
putblock and putreceipt in different method,I think sometimes they will be put in different db file

Reply:
blockHeightToFileKey -> heightToFilePrefix

bhf. -> hf.